docs/utilities/permissionapi.md

```diff
@@ -0,0 +1,55 @@
+PermissionAPI
+=============
+
+The PermissionAPI is a very basic implementation of a permission system.
+
+It's default implementation doesn't add propper permission handling though, but instead just has 3 "permission levels", (ALL = all players, OP = operators, NONE = no player).
```

Comment:
Ya... your pessimistic "the default implementation is shit" attitude isn't gonna fly. Its simplistic for a reason. Rewrite this and the several other lines like it to be less subjective.

Reply:
Oh, that wasn't my intention in the first place,
I just wrote it like that as most people know Permissions as something like you have with PEX,
and I didn't want to create any false assumptions about this. Hope the latest commit fixes that.